src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckClassVisitor.kt

```diff
@@ -76,27 +73,30 @@ internal class LincheckClassVisitor(
         if (access and ACC_NATIVE != 0) return mv
         if (instrumentationMode == STRESS) {
             return if (methodName != "<clinit>" && methodName != "<init>") {
-                CoroutineCancellabilitySupportMethodTransformer(mv, access, methodName, desc)
+                CoroutineCancellabilitySupportTransformer(mv, access, methodName, desc)
             } else {
                 mv
             }
         }
+        val createAdapter : (MethodVisitor) -> GeneratorAdapter = {
```

Comment:
Let's make it as a regular nested function:

```kotlin
fun createAdapter(mv: MethodVisitor) = GeneratorAdapter(mv, access, methodName, desc)
```

Reply:
Done